docs/src/development/shell-snippets.md

```diff
@@ -0,0 +1,18 @@
+# Shell Snippets in Documentation
+
+Snforge/sncast snippets and their outputs present in the docs are automatically run and tested. Some of them need to be configured with specific values to work correctly.
+
+## Snippet configuration
+
+To configure a snippet, you need to add a comment block right before it. The comment block should contain the configuration in JSON format. Example:
+
+```markdown
+<!-- { "package_name": "hello_starknet", "ignore_output": true } -->
+```
```

Comment:
Maybe a full example will be helpful here? wdyt?
```
<!-- { "package_name": "hello_starknet", "ignore_output": true } -->
```shell
$ sncast deploy \
...
...
``
```